tesseract/tesseract_planning/tesseract_process_planners/include/tesseract_process_planners/process_planner.h

```diff
@@ -51,7 +52,7 @@ class ProcessPlanner
    * @brief Given a tool path generate process specific plan
    * @param tool_path Process tool path
    */
-  virtual void setToolPath(const std::vector<std::vector<tesseract::tesseract_planning::WaypointPtr>>& tool_path) = 0;
+  virtual void setToolPath(const std::vector<std::vector<tesseract_motion_planners::WaypointPtr>>& tool_path) = 0;
```

Comment:
Somehow my build didn't catch this

Reply:
No library in the package was actually using this header. It was not until I created the test which instantiated all of the headers that this was exposed.